ci/run-ci.py

```diff
@@ -54,17 +57,18 @@
 SERVICE_ACCOUNT = 'substra-tests@substra-208412.iam.gserviceaccount.com'
 KEY_SERVICE_ACCOUNT = 'substra-208412-3be0df12d87a.json'
 
-SUBSTRA_TESTS_BRANCH = 'k8s-tasks'
-SUBSTRA_BACKEND_BRANCH = 'melloddy'
-HLF_K8S_BRANCH = 'master'
+SUBSTRA_TESTS_BRANCH = 'fix-concurrency'
+SUBSTRA_GIT_REF = 'master'
```

Comment:
BRANCH or GIT_REF ?

Reply:
SUBSTRA_GIT_REF according to the substra-tests Dockerfile, unless you want to change it there (and in the Makefile) too